### PR TITLE
DM-30058: Revise DMTN-093

### PR DESCRIPTION
**Pull request by ebellm**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -42,17 +42,23 @@ Avro schemas can be composed of nested sub-schemas under a top level namespace.
...
+
 Management and Evolution
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 Avro packets may be shipped with or without an embedded schema.
-Since the schema can be lengthy, and since each alert will be packaged for shipping separately, we expect to distribute alerts separately from the associated schema.
+Since the schema can be lengthy, and since each alert will be packaged for shipping separately, we expect to distribute alerts to community alert brokers without including the associated schema.
 
 The recipient of each packet requires a schema to interpret the data they have been sent.
 In the simplest model, the producer (the LSST Alert Distribution System) and the consumer (the remote broker or science user) simply agree on the schema prior to the start of the stream.
```
Comment (spenczar) [inline index.rst]: Perhaps remove `or science user` from this line, or revise all the way to `and the remote broker simply agree`.

File: index.rst [hunk 2]
```diff
@@ -127,214 +138,45 @@ maintained by Confluent Inc., the team that created Kafka.
...
-users connected to the LSST alert filtering system) is set at 100,
-meaning that the system is at least required to support the
-bandwidth needed for 100 filtered streams with 20 alerts per visit.
-Increasing the filtered alert limit could then mean that fewer
-users would be able to be supported due to networking limitations.
-DMS-REQ-0343 notes that the requirement could also be satisfied
-by cutting down the content of filtered alerts and potentially
-removing fields to decrease the size of filtered streams.
+Selected community alert brokers will receive the full LSST alert stream and provide a range of user tools to identify alerts of interest.
+We are currently evaluating technical approaches for on-project filtering of the alert stream for users with LSST Data Rights (see :cite:`RDO-013`).
```
Comment (spenczar) [inline index.rst]: What does "on-project" mean in this sentence?

File: index.rst [hunk 3]
```diff
@@ -383,6 +210,7 @@ week without reprocessing all alerts available since then.
 Daily topics also make expiring nights of data straightforward
 instead of ending up expiring data somewhere in the middle
 of the night.
+However, daily topics require more manual management by downstream consumers, and large numbers of Kafka topics can create stability issues.
```
Comment (spenczar) [inline index.rst:213]: Yeah, I might want to come back and argue against daily topics more thoroughly... I should do some writing on it to clarify my own thoughts.

File: index.rst [hunk 4]
```diff
@@ -25,8 +25,8 @@ Furthermore, image — or other — files can be embedded in Avro packets, maki
 Libraries for reading and writing Avro are available in many languages, including Python.
 
 Each alert is packaged as its own Avro packet, as opposed to wrapping groups of alerts per visit together.
+Packaging alerts separately allows brokers to take individual alerts as input and process each alert independently without having to repackage groups.
 As alerts are anticipated to arrive independently from the end of the Alert Generation Pipeline parallelized by CCD, the Kafka platform (see :ref:`alertDist`) acts as a cache before distribution, and individually packaged alerts makes this process simple.
```
Comment (spenczar) [inline index.rst]:
```suggestion
As alerts are anticipated to arrive independently from the end of the Alert Generation Pipeline parallelized by CCD, the Kafka platform (see :ref:`alertDist`) acts as a cache before distribution, and individually packaging alerts makes this process simple.
```

File: index.rst [hunk 5]
```diff
@@ -42,17 +42,23 @@ Avro schemas can be composed of nested sub-schemas under a top level namespace.
...
+During operations, users will access the production schemas using other tools (see :ref:`alertManagement`).
+
 .. _lsst/alert_packet: https://github.com/lsst/alert_packet
 
+.. _alertManagement:
+
 Management and Evolution
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 Avro packets may be shipped with or without an embedded schema.
```
Comment (spenczar) [inline index.rst]: This line is a little confusing. It sounds like a normative statement about what the project will do. I think it's intended to actually say "embedded schemas are technically possible in Avro but not required", or something?

Anyway, it's confusing and could use an edit.

**Review by spenczar — COMMENTED** [review]
This looks really good. I have a few minor nits to pick, but broadly seems right.

File: index.rst (hunk 2 above)
Comment (ebellm) [reply index.rst]: I'll revise to say "LSST-hosted."

File: index.rst (hunk 3 above)
Comment (ebellm) [reply index.rst:213]: Right, I thought we would want to discuss it.  I'll add a weasel sentence that we're still thinking about it.

File: index.rst (hunk 2 above)
Comment (spenczar) [reply index.rst]: Okay. I think DMTN-165 goes even further. "Alerts on your laptop" means *filtering* of the alert stream happens off of LSST-hosted computers.

I don't really think it's worth our time to back-and-forth this one too much though - LSST-hosted is fine too.

File: index.rst (hunk 5 above)
Comment (ebellm) [reply index.rst]: Understood--I took another crack at it.

**Review by spenczar — APPROVED** [review]
looks good!

File: index.rst (hunk 2 above)
Comment (ebellm) [reply index.rst]: Yes, but DMTN-165 is definitely a scope change--so unless/until that is approved by project-level CCB I don't want to lean too hard into it.

**Pull request merged**